specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/managedInstanceAdministrators.json

```diff
@@ -0,0 +1,420 @@
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete administrator of managed instance": {
+            "$ref": "./examples/ManagedInstanceAdministratorDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ManagedInstanceAdministratorListResult": {
+      "description": "A list of managed instance administrators.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedInstanceAdministrator"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ManagedInstanceAdministratorProperties": {
```

Comment:
>"ManagedInstanceAdministratorProperties": { [](start = 4, length = 43)

Are there any updatable properties on administrator resources ?.. are tags supported ?
If any of them is true, we need 'PATCH' as well, to be able to update them.

Reply:
No, there are no parameters that customer could change explicitly. Tags are not supported also.